backend/spec/requests/api/v1/project_developers_spec.rb

```diff
@@ -38,6 +38,91 @@
         end
       end
     end
+
+    post "Create new Project Developer" do
+      tags "Project Developers"
+      consumes "multipart/form-data"
+      produces "application/json"
+      security [csrf: [], cookie_auth: []]
+      parameter name: :project_developer_params, in: :formData, schema: {
+        type: :object,
+        properties: {
+          language: {type: :string, enum: Language::TYPES},
+          picture: {type: :file},
+          name: {type: :string},
+          about: {type: :string},
+          website: {type: :string},
+          linkedin: {type: :string},
+          facebook: {type: :string},
+          twitter: {type: :string},
+          instagram: {type: :string},
+          project_developer_type: {type: :string, enum: ProjectDeveloperType::TYPES},
+          entity_legal_registration_number: {type: :string},
+          mission: {type: :string},
+          "categories[]": {type: :array, items: {type: :string, enum: Category::TYPES}, collectionFormat: :multi},
```

Comment:
another rswag bug is here with `collectionFormat ` when used together with `formData `. It allows you to use multiple items from selectbox, but it sends it as `["value-1, value-2"]` <-- one string inside array (https://github.com/rswag/rswag/issues/498). Unfortunately I was not able to fix this, therefore only one item can be picked via rswag during testing of this.